.github/workflows/test_samba.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
       max-parallel: 5
       matrix:
-        python-version: [2.7]
+        python-version: [2.7, 3.5, 3.6, 3.7, 3.8]
```

Comment:
I recommend dropping 3.5 support.  It's not much supported anymore https://www.python.org/dev/peps/pep-0478/#id4